xml/day2_ui_documentation.xml

```diff
@@ -224,6 +224,85 @@
   </mediaobject>
  </figure>
 </section>
+<section xml:id="admin_ui_suse_cloud_services_model">
+ <title>Model</title>
+ <para>
+  The <guimenu>Model</guimenu> tab displays input models that
+  are deployed in the cloud and the model files associated with cloud. 
+  Model files listed can be modified.
+ </para>
+ <figure xml:id="admin_ui_cloud_services_model">
+  <title>&clm; Admin UI &suse; Service Model</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_model_0.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_model_0.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+  Clicking one of the listed files will open up the file editor where
+  changes can be made to the model file. Files that have
+  been edited but have not had their updates applied to the cloud
+  will be identified by an asterisk `*` in the file list.
+ </para>
+ <figure xml:id="admin_ui_cloud_services_model_editor">
+  <title>&clm; Admin UI &suse; Service Model Editor</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_model_1.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_model_1.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+  After editing the model file, click the <guimenu>Validate</guimenu>
+  button to validate changes. If validation is successful, <guimenu>Update</guimenu>
+  will be enabled. Click the <guimenu>Update</guimenu> button to deploy the changes to 
+  the cloud. Before start deployment, a confirmation dialog shows the choices of only
+  running config-processor-run.yml and ready-deployment.yml playbooks or running full
+  deployment. It also indicates the risk of updating the deployed cloud. 
+ </para>
+ <figure xml:id="admin_ui_cloud_services_model_confirmation">
+  <title>&clm; Admin UI &suse; Service Model Confirmation</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata fileref="clm_admin_services_model_2.png" width="75%"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="clm_admin_services_model_2.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <para>
+ Click <guimenu>Update<guimenu> to start deployment. The status of those changes will be
```

Comment:
the second <guimenu> here is the cause of the validation errors, its supposed to be `</guimenu>` ... missing the forwardslash

Reply:
I will edit and do a new PR this morning.